_posts/2020-05-04-report-cs-teaching-methods.md

```diff
@@ -0,0 +1,79 @@
+
+    ![Example 5 by 5 grid](https://storage.googleapis.com/cs-unplugged.appspot.com/static/img/topics/parity-cards-6x6-grid-step-1.png)
+
+    Step 1: Example layout of a 5 x 5 grid set up by the volunteer. [csunplugged.org](https://csunplugged.org/en/topics/error-detection-and-correction/unit-plan/parity-magic/)
+
+2. Then the instructor adds an additional row and column with "parity bits". The result should be a 6 x 6 with an even number of black tiles in each row and column.
+
+    ![adding a parity bit animation](https://storage.googleapis.com/cs-unplugged.appspot.com/static/img/topics/parity-cards.gif)
+
+    Step 2: Adding a parity bit to each row and column. [csunplugged.org](https://csunplugged.org/en/topics/error-detection-and-correction/unit-plan/parity-magic/)
+
+3. *With the instructor not looking*, the student flips one of the squares over, changing its color.
+
+4. The instructor then looks at the board and silently scans the black tiles in each row. Once an odd number of black tiles is found in a row, the columns can be scanned. When the corresponding column is found the (x,y) point of the flipped tile is known.
+
+Students then break into groups of 2 to 3, each group with its own set tile cards, and take turns flipping or finding the "erroneous" bits.
+
+# Challenges
+Due to the onset of the COVID-19 pandemic, hands-on group activties in a classroom are put in a different light. There is now a need to switch to online versions of the activies in order to perform the experiment in the fall.
+
+- **Problem:** How to develop online activies that aren't just coding exercises?
+    - One aspect of this research is to study non-coding activies. I want to test if teaching practical analogies of abstract digital concepts is worth doing.
+    
+    - Custom online activies can be produced but this may take lots of development time.
+    
+    - Fortunately, there is an interactive version of the sample [Card Flip Error Detection](https://csfieldguide.org.nz/en/interactives/parity/) activity.
+
+- **Problem:** How to engage students in the activity when the course is delivered asynchronously online?
+    - Part of the engament in the activities was planned to come from group dynamics. 
+        - 2 large groups of classmates racing against each other in a sorting activity
+        - Or many small groups performing the error detection activity
+    - Without 1 on 1 engagement between the instructor and students, the connections may not be made between the analogous activity and the algorithm or concept being taught.
+
+I do not have answers to these problems but will continue to study and work to figure out some workarounds or solutions.
+
+# Accomplishments
+Dr. Jamil Saquer has agreed to work with me on this research for Missouri State's CSC 131 course lab sections. This may not be likely in Fall 2020 since there is only 1 lab section offered and the COVID-19 requirements may move this section online.
```

Comment:
```suggestion
Dr. Jamil Saquer has agreed to work with me on this research for Missouri State's CSC 131 course lab sections. This may not be possible in Fall 2020 since there is only 1 lab section offered and the COVID-19 requirements may move this section online.
```